Source/Scene/Cesium3DTileset.js

```diff
@@ -70,6 +72,7 @@ define([
      * @param {Matrix4} [options.modelMatrix=Matrix4.IDENTITY] A 4x4 transformation matrix that transforms the tileset's root tile.
      * @param {Number} [options.maximumScreenSpaceError=16] The maximum screen-space error used to drive level-of-detail refinement.
      * @param {Boolean} [options.refineToVisible=false] Whether replacement refinement should refine when all visible children are ready. An experimental optimization.
+     * @param {Boolean} [options.useDynamicScreenSpaceError=false] Reduce the screen space error for tiles that are further away from the camera.
```

Comment:
We generally don't start variables with `use`, e.g., above is just `refineToVisible`, not `useRefineToVisible`, so this could just be `dynamicScreenSpaceError` or `scaleScreenSpaceErrorByDistance`.